opencti-platform/opencti-front/src/utils/hooks/useLocalStorage.ts

```diff
@@ -665,16 +665,14 @@ export const usePaginationLocalStorage = <U>(
       dispatch(`${key}_paginationStorage`, newValue);
     },
     handleAddFilterWithEmptyValue: (filter: Filter) => {
-      if (viewStorage?.filters) {
-        const { filters } = viewStorage;
-        const newValue = {
-          ...viewStorage,
-          filters: handleAddFilterWithEmptyValueUtil({ filters, filter }),
-          latestAddFilterId: filter.id,
-        };
-        setValue(newValue);
-        dispatch(`${key}_paginationStorage`, newValue);
-      }
+      const { filters } = viewStorage;
```

Comment:
why this change of logic ? just to understand, since it's a function used everywhere.

Reply:
Adrien has corrected this
it was blocking us because without a default filter value, it is completely blocked

Reply:
Yes, the whole point was to have consistency in that function : the goal is yo create an empty filter value but we assume we already have filters in our storage. 

So the change is simply to handle empty value in storage by the using the emptyFilterGroup constant (that we use for example in clearAllFilters)